services/cd-service/pkg/repository/repository.go

```diff
@@ -1027,24 +1034,25 @@ func (r *repository) ApplyTransformersInternal(ctx context.Context, transaction
 
 			//The ID of the transformer needs to be inserted into the json blob of the event_sourcing_light table,
 			// SO we can't read it after writing it
-			if r.DB.ShouldUseOtherTables() {
-				value, err := r.DB.DBDiscoverCurrentEsldID(ctx, transaction)
-				if err != nil {
-					fmt.Println(err)
-					return nil, nil, nil, &TransformerBatchApplyError{
-						TransformerError: err,
-						Index:            i,
-					}
-				}
-				var id db.TransformerID
-				if value == nil {
-					id = 1
-				} else {
-					//This is now guaranteed to be the next index
-					id = db.TransformerID(uint(*value) + 1)
-				}
-				t.SetEslID(id)
-			}
+			//if r.DB.ShouldUseOtherTables() {
```

Comment:
Please remove this commented section if it is not needed.

Reply:
done